layouts/soarian/r4/base/entities.html

```diff
@@ -0,0 +1,5 @@
+<h1 id="entities">Entities</h1>
+
+<p>Entities are used to track, relate, and locate things and people across the healthcare spectrum. They include things like Location, but also the Person resource. The Person resource is used to index a real world person providing a means to identify the same person across different healthcare organizations and roles.</p>
```

Comment:
Please add a comma after the phrase, "The Person resource is used to index a real world person"
This will make the sentence easier to read and understand.